### PR TITLE
correct other income text display on FA application review

### PR DESCRIPTION
**Pull request by mdkaraman**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

# What is the ticket # detailing the issue?

Ticket: RM-98782

# A brief description of the changes

Current behavior:
When an IAP applicant inputs Other Income, the system displays Other Deductions in the Application Summary.

New behavior:
When an IAP applicant inputs Other Income, the system displays Other taxable income in the Application Summary.

# Environment Variable

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. Please share the name of the variable below that would enable/disable the feature and which client it applies to.

Variable name: n/a

- [ ] DC
- [ ] ME

### CODE REVIEW

File: components/financial_assistance/app/views/financial_assistance/applications/raw_application.html.erb [hunk 1]
```diff
@@ -30,7 +30,7 @@
               <% income_and_deductions_for(applicant).each do |embedded_document| %>
                 <tr>
                   <td> <%= applicant.full_name %> </td>
-                  <td> <%= l10n("faa.income.#{embedded_document.kind}") %> </td>
+                  <td> <%= l10n("faa.#{embedded_document.class.name.demodulize.downcase}.#{embedded_document.kind}") %> </td>
```
Comment (scaustin34) [inline components/financial_assistance/app/views/financial_assistance/applications/raw_application.html.erb:33]: TIL 'demodulize'

**Review by scaustin34 — APPROVED** [review]
Looks good to me 🤘

File: components/financial_assistance/app/views/financial_assistance/applications/raw_application.html.erb (hunk 1 above)
Comment (scaustin34) [reply components/financial_assistance/app/views/financial_assistance/applications/raw_application.html.erb:33]: i wish you could leave comments without them needing to be 'resolved' 
my bad

**Review by polographer — APPROVED** [review]
lgtm!

**Pull request merged**